uktrade/data-hub-frontend
# Remove formidable dependency (and reactify proposition evidence form)

## Description of change

We currently have a **critical** vulnerability relating to the `formidable` dependency. As the latest version of `formidable` isn't compatible with the legacy code that's currently using it and because it's only being used in proposition evidence form I have taken the opportunity to reactify this area of the codebase. This has allowed the dependency to be removed.

I have also removed the legacy 'Complete proposition' form as this wasn't being used and also allowed the removal of a large amount of dead code.

## Test instructions

Go to a proposition and upload an evidence document. Everything should work as normal.

## Screenshots

<img width="584" alt="Screenshot 2024-04-23 at 12 44 21" src="https://github.com/uktrade/data-hub-frontend/assets/36161814/fe6daa2a-3ba9-4be2-b04b-3b09a97122ce">


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
